### PR TITLE
gifcap 1.0.3 (new formula)

### PR DESCRIPTION
**Pull request by benjamin-bader**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

A small program to record GIF videos from an Android device.  It's quite useful during development; our team uses it regularly to capture interesting app behavior, drive design discussions, etc.

Another Homebrew formula that is similar in scope and utility is `pidcat`; we hope that our contribution is acceptable.  If it is deemed too specialized or small, I'll totally understand.  Thanks for considering it!

### CODE REVIEW

File: Formula/gifcap.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  desc "Capture video from an Android device and make a gif"
+  homepage "https://github.com/outlook/gifcap"
+  url "https://github.com/outlook/gifcap/archive/1.0.3.tar.gz"
+  sha256 "43b5f477b11f479ce2f4621e0e2c90ed5bc4563fc5f1dcbabfa6fe72e02454c9"
+  head "https://github.com/outlook/gifcap.git"
+
+  bottle :unneeded
+
+  depends_on "ffmpeg"
+  depends_on "android-sdk" => :optional
```
Comment (woodruffw) [inline Formula/gifcap.rb]: Is this really optional? My guess is that `adb` is necessary for this, right?

Comment (woodruffw) [inline Formula/gifcap.rb]: (Ignore me if the idea is that users might already have the SDK installed, and might not want it through `brew`)

Comment (benjamin-bader) [inline Formula/gifcap.rb]: You're correct on both counts; I *believe* that most people either manually manage their own SDK, or just use Android Studio's copy.

Comment (woodruffw) [inline Formula/gifcap.rb]: Thanks for confirming; I'm not an Android dev :smile:

**Comment by woodruffw** [comment]
This looks good to me. Will leave open for a bit to give other maintainers time to provide feedback.

**Comment by benjamin-bader** [comment]
Thanks, @woodruffw!  I appreciate your time & attention.

**Comment by woodruffw** [comment]
Thanks for contributing, @benjamin-bader!

**Pull request closed without merging**